apps/admin/backend/src/store.ts

```diff
@@ -1250,6 +1258,37 @@ export class Store {
     return sheetNumber ? { type: 'hmpb', sheetNumber } : { type: 'bmd' };
   }
 
+  private parseVotesWithAdjudications({
+    votesString,
+    adjudicationsString,
+  }: {
+    votesString: string;
+    adjudicationsString: string | null;
+  }): Tabulation.Votes {
+    const votes = JSON.parse(votesString) as Tabulation.Votes;
+    if (!adjudicationsString) return votes;
+
+    const adjudications = JSON.parse(
+      adjudicationsString
+    ) as CastVoteRecordVoteAdjudication[];
+
+    for (const adjudication of adjudications) {
+      const currentContestVotes =
+        votes[adjudication.contestId] ?? /* c8 ignore next 1 */ [];
```

Comment:
It should be, but the testing infrastructure I have for mocking specific cast vote records does not have this guarantee. Because our fixtures don't have `gridLayouts`, I don't have a good means to make those mocks accurate, so leaving for now.